page.go

```diff
@@ -27,24 +50,67 @@ func (p Page) Offset() int {
 	return (p.Number - 1) * p.Size
 }
 
+// Next returns a page with the number advanced by one
+func (p Page) Next() Page {
+	return Page{
+		Number:      p.Number + 1,
+		Size:        p.Size,
+		ResultCount: p.ResultCount,
+		NextURL:     p.NextURL,
+		PrevURL:     p.PrevURL,
+	}
+}
+
+// Prev returns a page with the number decremented by 1
+func (p Page) Prev() Page {
+	return Page{
+		Number:      p.Number - 1,
+		Size:        p.Size,
+		ResultCount: p.ResultCount,
+		NextURL:     p.NextURL,
+		PrevURL:     p.PrevURL,
+	}
+}
+
+// SetQueryParams adds pagination info to a url as query parameters
+func (p Page) SetQueryParams(u *url.URL) *url.URL {
```

Comment:
This only adds "page" as a param, if "PageSize" isn't the default "size" then we should be able to add that as a param as well.